tools/setup/optimize-svg

```diff
@@ -36,23 +39,37 @@ while true; do
     esac
 done
 
-if [ "$#" -gt 0 ]; then
+if [ "$#" -gt 1 ]; then
     usage >&2
     exit 1
 fi
 
 ZULIP_PATH="$(readlink -f "$(dirname "$0")"/../..)"
 PNPM="/usr/local/bin/pnpm"
+BASE_PATH="static/images/integrations/logos"
+
+if [ "$#" -eq 0 ]; then
+    FILE_PATH="$BASE_PATH"
+    SVGO_FLAG="-f"
+else
+    FILE_PATH="$BASE_PATH/$1"
+    SVGO_FLAG=""
+fi
 
 if [ -n "$CHECK_UNOPTIMIZED" ]; then
-    if [ "$(node_modules/.bin/svgo -f static/images/integrations/logos | grep -o '\.[0-9]% = ' | wc -l)" -ge 1 ]; then
-        echo "ERROR: svgo detected unoptimized SVG files in the \`static/images/integrations/logos\` folder." 1>&2
+    TEMP_PATH=$(mktemp -d svgo_temp_XXXXXX)
+    RESULT=$("$PNPM" exec svgo -o "$TEMP_PATH" $SVGO_FLAG "$FILE_PATH" | grep -o '\.[0-9]% = ' | wc -l)
+    rm -rf "$TEMP_PATH"
+    if [ "$RESULT" -ge 1 ]; then
+        echo "ERROR: svgo detected unoptimized SVG file(s)." 1>&2
         echo "Please run tools/setup/optimize-svg and commit the file changes to optimize them."
         exit 1
     else
-        echo "SUCCESS: SVG files in static/images/integrations/logos are all optimized!"
```

Comment:
Not echoing the path allows us to avoid redundancies when introducing the new argument `filename`.
And the path is still easily discoverable using `--help`.